1.2/Dockerfile

```diff
@@ -12,7 +12,13 @@ ENV NPM_VERSION ${NPM_VERSION}
 LABEL io.k8s.description="Platform for building and running a phoenix app" \
       io.k8s.display-name="build-phoenix" \
       io.openshift.expose-services="4000:http" \
-      io.openshift.tags="builder,elixir,phoenix"
+      io.openshift.tags="builder,elixir,phoenix" \
+      com.redhat.dev-mode="DEV_MODE:false" \
```

Comment:
What does that label do?

Reply:
Removed in 551f804, I did accidentally copy that.